core/src/main/java/me/leoko/advancedban/utils/tabcompletion/PunishmentTabCompleter.java

```diff
@@ -46,14 +50,20 @@ public List<String> onTabComplete(Object user, String[] args) {
                     suggestions.add(amount + unit);
                 }
             }
-            for (String layout : methodInterface.getKeys(methodInterface.getLayouts(), "Time")) {
-                suggestions.add("#"+layout);
+            if (methodInterface.getBoolean(methodInterface.getLayouts(), "Use Time Layouts")) {
+                for (String layout : methodInterface.getKeys(methodInterface.getLayouts(), "Time")) {
+                    suggestions.add("#"+layout);
+                }
             }
+
         } else if((temporary && args.length == 3) || args.length == 2) {
-            suggestions.add("Reason...");
-            for (String layout : methodInterface.getKeys(methodInterface.getLayouts(), "Message")) {
-                suggestions.add("@"+layout);
+            if (methodInterface.getBoolean(methodInterface.getLayouts(), "Use Message Layouts", true)) {
+                suggestions.add("Reason...");
```

Comment:
I'm not sure whether "Reason..." is related to layouts in particular, or if it's supposed to represent reasons more generally.

Reply:
Shouldn't be related to it as far as I'm seeing, this seems to act as a placeholder (such as the placeholder "[Name]" on the first argument or Xh Xm etc on temporary punishments). So I'd say this should still be included when disabling layouts.